rna-transcription.json

```diff
@@ -0,0 +1,105 @@
+{
+   "#": [
+     "Language implementations vary on the issue of missing input variables.",
+     "In this set of test cases, the input variable is left out of the definition,",
+     "and the generator would need to handle the language implementation."
```

Comment:
Let's also mention that -1 represents an error case, each language must implement it in a way that makes sense for that language.